### PR TITLE
service network command

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by pwright** [comment]
@nluaces there's a preview at https://622f2f5d08f8390008b95393--skupper.netlify.app/skupper/latest/cli/index.html#network-service-

**Review by nluaces — APPROVED** [review]

File: modules/ROOT/partials/network-status.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+│  URL: 10.96.146.236 <2>
+│  name: west <3>
+│  namespace: west
+│  version: 0.8.6 <4>
+│  ╰─ Services:
+│     ╰─ name: hello-world-backend <5>
+│        address: hello-world-backend: 8080 <6>
+│        protocol: tcp <7>
+╰─ [remote] bca99d1 - east <8> 
+   URL:
```
Comment (ajssmith) [inline modules/ROOT/partials/network-status.adoc:39]: Is it worth noting a blank URL occurs when "--ingress none" or iiuc when in ''--router-mode" edge"?

File: modules/ROOT/partials/network-status.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      ╰─ name: hello-world-backend
+         address: hello-world-backend: 8080
+         protocol: tcp
+         ╰─ Targets:
+            ╰─ name: hello-world-backend-7dfb45b98d-mhskw <10>
+----
+
+<1> The unique identifier of the site associated with the current context, that is, the `west` namespace
+
+<2> The URL of the Skupper router.
```
Comment (ajssmith) [inline modules/ROOT/partials/network-status.adoc:54]: iiuc, the URL is the inter-router-link listener as opposed to the console url that skupper status returns.

File: modules/ROOT/partials/network-status.adoc (hunk 1 above)
Comment (pwright) [reply modules/ROOT/partials/network-status.adoc:39]: added note at end to explain this with '--ingress none'.. we don't mention edge router elsewhere, so avoiding that

**Review by ajssmith — APPROVED** [review]
lgtm, thanks for the updates

**Pull request merged**